Jellyfin.Plugin.TelegramNotifier/Jellyfin.Plugin.TelegramNotifier.csproj

```diff
@@ -12,7 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.9.7" />
```

Comment:
```10.*-*``` was used for a better compatibility across the servers and the shared libraries.